catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/src/main/webapp/js/view/Segment.view.js

```diff
@@ -62,15 +62,16 @@ define([
 
                 var standardFields = [];
                 var advancedFields = [];
-                this.model.get('fields').forEach( function (field) {
+                this.model.get('fields').forEach(function (field) {
+                    field.set('identityNode', this.model.get('identityNode'));
                     if (!field.get('custom') && !field.get('advanced')) {
                         standardFields.push(field);
                     }
                     if(field.get('advanced')){
                         advancedFields.push(field);
                         this.addvancedFields = true;
                     }
-                });
+                }.bind(this));
```

Comment:
I might be mis-remembering, but I'm pretty sure you can also just do `forEach(function() {...}, this);`